ecr-login/api/client.go

```diff
@@ -29,22 +29,22 @@ import (
 const proxyEndpointScheme = "https://"
 
 type Client interface {
-	GetCredentials(registry, image string) (string, string, error)
+	GetCredentials(registry, image string) (string, string, string, error)
 }
 type defaultClient struct {
 	ecrClient       ecriface.ECRAPI
 	credentialCache cache.CredentialsCache
 }
 
-func (self *defaultClient) GetCredentials(registry, image string) (string, string, error) {
+func (self *defaultClient) GetCredentials(registry, image string) (string, string, string, error) {
```

Comment:
Can you add a godoc comment here explaining the return parameters?

I'm almost convinced that we should move to named returns here, since there are now three different strings.